pkg/master/master.go

```diff
@@ -777,7 +778,13 @@ func (m *Master) api_v1() *apiserver.APIGroupVersion {
 
 // expapi returns the resources and codec for the experimental api
 func (m *Master) expapi(c *Config) *apiserver.APIGroupVersion {
-	storage := map[string]rest.Storage{}
+
+	controllerStorage := expcontrolleretcd.NewStorage(c.DatabaseStorage, c.KubeletClient)
+	storage := map[string]rest.Storage{
+		strings.ToLower("replicationControllers"):        controllerStorage.ReplicationController,
```

Comment:
ToLower - so that the next person who adds something here knows that it should all in lowercase. 
CamelCase - followed the pattern from https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/master/master.go#L477. Can put all in lowercase if you prefer.